### PR TITLE
Add test case for delete term meta module

### LINKED ISSUE #350: Delete Term meta - Add tests
- [x] Delete default taxonomy term meta with equal value
- [x] Delete default taxonomy term meta with not equal value
- [ ] Delete custom taxomomy term meta with equal value
- [ ] Delete custom taxomomy term meta with not equal value

### PR DESCRIPTION
**Pull request by rajanit2000**
Fix #350

Blocked by #247

### CODE REVIEW

**Comment by rajanit2000** [comment]
@sudar development finished ready to review

File: tests/wp-unit/include/Core/Metas/Modules/DeleteTermMetaModuleTest.php [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+	public function setUp() {
+		parent::setUp();
+
+		$this->module = new DeleteTermMetaModule();
+	}
+
+	/**
+	 * Add to test delete default taxonomy term meta with equal value.
+	 */
+	public function test_that_delete_default_taxonomy_term_meta_with_equal_value() {
```
Comment (sudar) [inline tests/wp-unit/include/Core/Metas/Modules/DeleteTermMetaModuleTest.php]: Function name is not following the `test_that_something_can_be_deleted` format.

Change it for all the methods.

File: tests/wp-unit/include/Core/Metas/Modules/DeleteTermMetaModuleTest.php [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+		// call our method.
+		$delete_options = array(
+			'term'             => $term_array['term_id'],
+			'term_meta'        => $meta_key,
+			'term_meta_value'  => $meta_value,
+			'term_meta_option' => 'equal',
+		);
+
+		$meta_deleted = $this->module->delete( $delete_options );
```
Comment (sudar) [inline tests/wp-unit/include/Core/Metas/Modules/DeleteTermMetaModuleTest.php]: You also need to add some terms that won't be deleted in the test cases.

File: tests/wp-unit/include/Core/Metas/Modules/DeleteTermMetaModuleTest.php (hunk 1 above)
Comment (rajanit2000) [reply tests/wp-unit/include/Core/Metas/Modules/DeleteTermMetaModuleTest.php]: Fixed

File: tests/wp-unit/include/Core/Metas/Modules/DeleteTermMetaModuleTest.php (hunk 2 above)
Comment (rajanit2000) [reply tests/wp-unit/include/Core/Metas/Modules/DeleteTermMetaModuleTest.php]: Fixed

File: tests/wp-unit/include/Core/Metas/Modules/DeleteTermMetaModuleTest.php [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+
+		// Assert that post meta deleted.
+		$this->assertEquals( 1, $meta_deleted );
+
+		$meta = get_term_meta( $term_array['term_id'], 'another' );
```
Comment (sudar) [inline tests/wp-unit/include/Core/Metas/Modules/DeleteTermMetaModuleTest.php]: In `add_term_meta` you are using the value `Another` and here you are checking for `another`.

Make sure you store the value in a variable before accessing it.

This problem is happening in all test cases. Fix it in all places.

Comment (rajanit2000) [reply tests/wp-unit/include/Core/Metas/Modules/DeleteTermMetaModuleTest.php]: Fixed

**Comment by rajanit2000** [comment]
@sudar its ready to review

**Pull request merged**